bmoreonrails/rails_tutorial
# Locally sourced images

#### What does this PR do?
Makes local copies of images that had been tagged with `"We should serve this locally"`.
#### Where should the reviewers start?
There should be no tricky changes, just replacing internet links with local links.
#### How should this be manually tested?
1. Open a browser's web console to the network tab.
1. Browse around the site.
1. Observe fewer requests for remote images than we have now.
#### Any background context you want to provide?
#### What are the relevant tickets?
#63 
#### Screenshots?
N/A
#### Questions:
None
